libsql-server/src/main.rs

```diff
@@ -228,6 +228,9 @@ struct Cli {
     /// encryption_key for encryption at rest
     #[clap(long, env = "SQLD_ENCRYPTION_KEY")]
     encryption_key: Option<bytes::Bytes>,
+    /// encryption_key for encryption at rest
+    #[clap(long, env = "SQLD_BLOCK_VACUUM")]
+    block_vacuum: bool,
```

Comment:
Doesn't it make it default to `false`, unless you specify otherwise? It may make sense to introduce a `SQLD_ALLOW_USER_VACUUM` and make it default to false everywhere, and that would work naturally for all kinds of libraries, rpc, serializers, etc. Main risk of accidentally allowing vacuum here is that it's a very I/O heavy operation, and we should prefer to always prevent users from issuing it themselves, or at least throttle it